app/deed/server.py

```diff
@@ -0,0 +1,21 @@
+from flask import jsonify
+
+
+def register_routes(blueprint):
+    @blueprint.route('/view-deed', methods=['GET'])
+    def get_deed():
+
+        result = {
+            "deed_id": "123456",
+            "provision_1": "This Mortgage Deed incorporates the Lenders Mortgage\
```

Comment:
Can you turn this in to an array? feels like we are likely to get more than or less than 3 provisions.
```
provisions: [
  "blah",
  "blargh",
  "bloz"
]
```